sdk/servicebus/azure-servicebus/azure/servicebus/_base_handler.py

```diff
@@ -330,17 +330,7 @@ def _convert_connection_string_to_kwargs(
 
         return kwargs
 
-    def __enter__(self):
```

Comment:
was this deleted by mistake?

Reply:
no, the typechecker was detecting the return type as BaseHandler when the subclasses used the context manager, instead of "SBSender"/"Receiver". So moved this to the subclasses, as per: https://github.com/python/mypy/issues/4678#issuecomment-370303827